doc/elements-confidential-transactions.md

```diff
@@ -0,0 +1,480 @@
+an issuance input. This key is required when using reissuance tokens. 
+
+`dumpmasterblindingkey`
+Reveals the wallet’s master blinding key from which all blinding keys for generated addresses are derived.  See SLIP-007.
+
+`importaddress`
+A confidential address may be imported at any time. However, in order
+to unblind outputs for a confidential address, it is necessary to also
+import the blinding key for that address’ confidential_key. See
+the `importblindingkey` RPC.
+
+`importblindingkey`
+Imports the private blinding key associated with an address.
+
+`importissuanceblindingkey`
+Imports a private blinding key that may be used to unblind the amounts
+on an issuance input, or reissue additional amounts of an asset (using
+reissuance tokens). 
+
+`importmasterblindingkey`
+***Use with caution!*** Importing a master blinding key into a wallet will
+erase the existing master blinding key, potentially making all
+addresses owned by that wallet unspendable.
+
+
+
+### Wallet - Transactions
+
+`blindrawtransaction`
+Blinds the outputs of a raw transaction (as might be created by `createrawtransaction`)
+
+
+`unblindrawtransaction`
+Unblinds a raw transaction, using addresses and blinding keys known to
+the wallet (they must have been previously imported, see "Third-party
+Unblinding" below.
+
+
+
+## Third-party Unblinding (Watch-only wallets)
+
+A third-party may be granted the ability to unblind the amounts and
+assets in a confidential transaction, without being able to spend the
+transaction’s UTXOs. Using Elements, the third-party would create a
+"watch-only wallet" for the addresses in question, and import the
+private blinding keys for those addresses.  
+
+Let’s suppose that Alice has sent a confidential transaction to
+Bob. Bob wants Victor to be able to see what and how much was sent. 
+
+Victor, with Bob’s help, creates a watch-only wallet in Elements:
+
+1. for each address of interest A, Victor runs the RPC `importaddress
+<A>`
+
+2. Bob exports the blinding key for address A using `dumpblindingkey
+<A>` and shares it with Victor.
+
+3. Victor imports the blinding key for the address using `importblindingkey <A> <blinding key>`
+
+Once the blinding key is imported, the Elements wallet will treat the
+Confidential address address as watch-only, and its outputs will be
+visible in transaction details and in the wallet balance.  
+
+Please note that if Bob reuses an address A, Victor will also be able
+to see the amounts and values in any transaction sending to A. 
+
+Alternatively, a watch-only wallet may import the master blinding key
+of another wallet. The watch-only wallet would then be able to view
+the UTXOs for any confidential address created by the original
+wallet. 
+
+Elements is not required to unblind a confidential transaction. Given
+the confidential address’ blinding key, the rangeproof for the output
+can be rewound, revealing the blinding factors and actual amounts and
+assets that were committed to.  
```

Comment:
What are these two sentences really communicating? If you're not using Elements, how else would you do it? (other than with a pen and paper)

It sounds like you might really just be trying to say the following:
```
Anyone with the blinding key for an output's confidential address can rewind the range proof and reveal the blinding factors, along with the actual amounts and assets that were committed to.
```